.github/workflows/latest_dependency_checker.yaml

```diff
@@ -24,7 +24,6 @@ jobs:
       run: |
         python -m virtualenv venv_core
         source venv_core/bin/activate
-        python -m pip install --upgrade pip
```

Comment:
why are we removing this?

Reply:
It is run earlier:
https://github.com/alteryx/woodwork/blob/main/.github/workflows/latest_dependency_checker.yaml#L21